src/main/java/com/vladyslavpalamarchuk/trackmycoin/service/NotificationService.java

```diff
@@ -0,0 +1,26 @@
+package com.vladyslavpalamarchuk.trackmycoin.service;
+
+import com.vladyslavpalamarchuk.trackmycoin.adaptors.persistence.MonitoringRepository;
+import com.vladyslavpalamarchuk.trackmycoin.adaptors.telegram.TelegramBotClient;
+import com.vladyslavpalamarchuk.trackmycoin.domain.Monitoring;
+import jakarta.transaction.Transactional;
+import java.math.BigDecimal;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+class NotificationService {
```

Comment:
Why do we need this service if sometimes we are using directly telegramBotClient and now via this service. I don't understand